en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -0,0 +1,107 @@
+# Configure a Global Key Manager
+
+This document outlines the steps to enable cross-tenant key manager support in WSO2 API Manager (API-M). It addresses the challenge faced when applications in one tenant need to invoke APIs published in a different tenant, resulting in HTTP 401 errors even with valid tokens. The proposed solution involves configuring a global key manager accessible across all tenants.
```

Comment:
instead of "applications in one tenant" can we say keys generated from a key manager of one tenant? Or some thing that gives that idea. This is because we currently generate separate keys for the same application.

Reply:
This document outlines the steps to enable cross-tenant key manager support in WSO2 API Manager (API-M). It addresses the challenge of requiring to generate tokens on a particular tenant's key manager in order to invoke an API belonging to that tenant. The proposed solution involves configuring a global key manager accessible across all tenants. With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.

WDYT?